quantum/encoder.c

```diff
@@ -130,12 +146,12 @@ void encoder_init(void) {
 #endif // defined(SPLIT_KEYBOARD) && defined(ENCODER_RESOLUTIONS)
 
     for (uint8_t i = 0; i < thisCount; i++) {
-        setPinInputHigh(encoders_pad_a[i]);
-        setPinInputHigh(encoders_pad_b[i]);
+        encoder_init_state(encoders_pad_a[i]);
```

Comment:
You sure this has been tested? As the interface takes index, but is passing through a  `pin_t`.